packages/graphqlgen/src/tests/fixtures/scalar/flow-types.js

```diff
@@ -0,0 +1,11 @@
+//@flow
+
+export interface Context {
+  db: any
```

Comment:
Why `any`?

Reply:
Because we somehow decided to always provide a context to the tests, but there is no way to type `db` properly.

There should just be only one test making sure that `Context` is rendered properly, and the other ones should just not provide any context at all